src/app/content/hooks/locationChange.spec.ts

```diff
@@ -102,7 +102,7 @@ describe('locationChange', () => {
 
   it('doesn\'t load page if its already loaded', () => {
     localState.page = {
-      id: 'pageId',
+      shortId: 'pageId',
```

Comment:
Since we use many identifiers in the system (uuid, shortId, uuid+ver, shortId+ver, slug) and they all have the same type (`string`) would it make sense to rename the field to help show which identifier is being used?

Reply:
the trouble with shortId is that in the full payload it is only the id, but in the tree it is shortId + version. currently the only place i care about this is in ContentLink but i could probably push it down to ArchiveLoader if you want